src/apigateway/commands/copyUrl.ts

```diff
@@ -61,6 +63,7 @@ export async function copyUrlCommand(
         window.showInformationMessage(
             localize('AWS.apig.copyUrlNoStages', "Failed to copy URL because '{0}' has no stages", node.name)
         )
+        recordApigatewayCopyUrl({ result: 'Failed' })
```

Comment:
if we get here it's most likely a user-issue, should we still consider this a fail?

Reply:
We currently don't disambiguate this anywhere else, and this is a rare instance where it's easy to disambiguate. I think we should leave fail in for now and if we ever disambiguate we can do it then along with everything else.